package/endpoint/data_stream/policy/fields/fields.yml

```
@@ -245,6 +245,23 @@
       ignore_above: 1024
       description: the overall status of event collection, this is correlated to the status of concerned actions  but not a simple sum of the actions
       default_field: false
+    - name: policy.applied.response.configurations.firewall_anti_tamper
+      level: custom
+      type: object
+      description: overall firewall_anti_tamper configuration and status of the applied policy
```

```suggestion
      description: overall firewall anti-tamper configuration and status of the applied policy
```

Looking at other descriptions, we never use variable (node) names to explain the variable

---

Good spot, it looks like `behavior_protection` and `memory_protection` use the same format at this, but most use plain english. I'll update this to "firewall anti-tamper".

---

Updated, thanks 👍